kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -901,6 +903,12 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help='Rekompile foundry.k even if kompiled definition already exists.',
     )
+    foundry_kompile.add_argument(
+        '--kompiled-dir',
+        dest='kompiled_dir',
+        type=str,
+        help='Folder (under [foundry-project-root]/out/) to output as kompiled directory.',
+    )
```

Comment:
Why is this option added? Was it something that seemed needed before, but now you don't?

Reply:
It was used when I was testing the lemma inclusion in a separate kompiled directory. It's no longer needed because testing lemmas are included in the main kompilation now.